assets/src/apps/authoring/components/AdaptivityEditor/ActionNavigationEditor.tsx

```diff
@@ -42,22 +60,41 @@ const ActionNavigationEditor: React.FC<ActionNavigationEditorProps> = (props) =>
           onBlur={(e) => handleTargetChange(e)}
           title={target}
         />
+        <div className="dropdown dropup adaptivityDropdown">
```

Comment:
what is preventing us from just doing `<ScreenDropdownTemplate />` here?

Reply:
<ScreenDropdownTemplate> renders the dropdown control where as in the ticket it was mentioned that there should be a provistion to enter the text value also. So i thought it would be bette to use just <ScreenDropdown> which will render the toggle button and a dropdown